tapdb/universe_forest.go

```diff
@@ -125,3 +130,87 @@ func (b *BaseUniverseForest) RootNodes(
 
 	return uniRoots, nil
 }
+
+// RegisterIssuance inserts a new minting leaf within the multiverse tree and
+// the universe tree that corresponds to the given base key.
+func (b *BaseUniverseForest) RegisterIssuance(ctx context.Context,
+	id universe.Identifier, key universe.BaseKey,
+	leaf *universe.MintingLeaf,
+	metaReveal *proof.MetaReveal) (*universe.IssuanceProof, error) {
+
+	var (
+		writeTx       BaseUniverseForestOptions
+		issuanceProof *universe.IssuanceProof
+	)
+
+	dbErr := b.db.ExecTx(
+		ctx, &writeTx, func(dbTx BaseUniverseForestStore) error {
```

Comment:
style nit: could extract this closure into a named one defined before the call to `b.db.ExecTx()` which would save us one level of indentation for the whole block.